en/components/grids_templates/advanced_filtering.md

```diff
@@ -0,0 +1,457 @@
+---
+title: Angular Grid Advanced Filtering | Ignite UI for Angular | infragistics
+_description: Learn how to configure advanced filter of data with the Angular Material table. The grid advanced filtering is more convenient and engaging than ever.
+_keywords: advanced filter, igniteui for angular, infragistics
+---
+
+### @@igComponent Advanced Filtering
+
+The grid Advanced filtering provides a dialog which allows creating groups of filtering conditions across all columns for any Angular Material table like the @@igComponent.
```

Comment:
The ~~grid~~ Advanced filtering
The **@@igComponent** Advanced filtering provides a dialog which allows **the creation of groups with filtering conditions**